### PR TITLE
chore: fix back button for onboarding

### PR DESCRIPTION
**Pull request by adamgrharvey**
Changes back button on `onboarding/SetupComplete` slide to go back 2 slides instead of 1 slide. 

If 'I Already Have My Credential' is selected on the `onboarding/SetupConnection` slide, the credential issuance is skipped. If you then go back to that credential connection page, the demo crashes with server error 500. By going back 2 slides, we bypass this connection request, and the demo continues to work.

### CODE REVIEW

**Comment by adamgrharvey** [comment]
@weslord

File: web/src/pages/onboarding/OnboardingContainer.tsx [hunk 1]
```diff
@@ -76,6 +76,9 @@ export const OnboardingContainer: React.FC<Props> = ({
     removeOnboardingProgress(dispatch, onboardingStep, currentCharacter)
   }
 
+  const jumpPrevOnboardingPage = () => {
+    removeOnboardingProgress(dispatch, onboardingStep, currentCharacter, 2)
+  }
```
Comment (weslord) [inline web/src/pages/onboarding/OnboardingContainer.tsx:82]: Please add inline comments with the rationale for this, will not be self-evident to future generations.

File: web/src/pages/onboarding/components/OnboardingBottomNav.tsx [hunk 2]
```diff
@@ -44,7 +46,11 @@ export const OnboardingBottomNav: React.FC<Props> = ({
       className="flex w-full justify-between mb-4 h-8 self-end select-none"
     >
       <div className="flex self-center">
-        <BackButton onClick={removeOnboardingStep} disabled={backDisabled} data-cy="prev-onboarding-step" />
+        <BackButton
+          onClick={isCompleted ? removeTwoOnboardingStep : removeOnboardingStep}
+          disabled={backDisabled}
+          data-cy="prev-onboarding-step"
+        />
```
Comment (weslord) [inline web/src/pages/onboarding/components/OnboardingBottomNav.tsx]: Please add inline comments with the rationale for this, will not be self-evident to future generations.

**Review by weslord — CHANGES_REQUESTED** [review]

File: web/src/pages/onboarding/OnboardingContainer.tsx (hunk 1 above)
Comment (adamgrharvey) [reply web/src/pages/onboarding/OnboardingContainer.tsx:82]: Added.

File: web/src/pages/onboarding/components/OnboardingBottomNav.tsx (hunk 2 above)
Comment (adamgrharvey) [reply web/src/pages/onboarding/components/OnboardingBottomNav.tsx]: Added.

Comment (weslord) [reply web/src/pages/onboarding/components/OnboardingBottomNav.tsx]: I'd put the comment right next to the part that is likely to cause confusion (the one-step two-step conditional).

Comment (adamgrharvey) [reply web/src/pages/onboarding/components/OnboardingBottomNav.tsx]: Done

**Review by weslord — APPROVED** [review]

**Review by jleach — APPROVED** [review]

**Pull request merged**